editor/editor_settings.cpp

```diff
@@ -895,6 +896,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	EDITOR_SETTING(Variant::FLOAT, PROPERTY_HINT_RANGE, "editors/polygon_editor/auto_bake_delay", 1.5, "-1.0,10.0,0.01");
 
 	// Animation
+	_initial_set("editors/animation/default_animation_step", Animation::DEFAULT_STEP);
```

Comment:
However, I think you should consider adding two additional items: FPS/Second mode and FPS compatible mode in second mode. If not added, the default config of those should be documented.

Reply:
Can you elaborate what these settings would do exactly?
"FPS/Second mode" sounds like we should just remember the option in AnimationPlayer.

Reply:
Does that mean that default_animation_step is always seconds? If so, that would make sense. However, I think it would be better to have a note in the documentation that says, “If you want 10 FPS to be the default, you need to set 0.1".

For FPS compatible mode, see https://github.com/godotengine/godot/pull/99013. This is an option that internally sets whether it snaps to FPS or not.

Reply:
The setting will affect `set_step()`, which I think is always seconds?

> For FPS compatible mode

Looks like another option to remember state.
Do you think it's better to store it globally or per project?

Reply:
> The setting will affect set_step(), which I think is always seconds?

I remember that `animation->set_step()` is always seconds even if it is in FPS mode. It is different from `set_step()` in the editor's spin box.

> Looks like another option to remember state.
> Do you think it's better to store it globally or per project?

I believe it should be just next to `default_animation_step` setting.